### PR TITLE
Refractor codebase a TON

### PR DESCRIPTION
**Pull request by greysilly7**
100% not ready for merge, ned to test the scam detention doo hicky

### CODE REVIEW

**Comment by greysilly7** [comment]
Fixed the Temp Banning on SCam Link

**Comment by greysilly7** [comment]
Everything should work a-okay

**Comment by greysilly7** [comment]
Everything complained about should be fixed if I missed something just say "fix x thing"

File: bot.js [hunk 1]
```diff
@@ -1,214 +1,202 @@
...
-const fs = require("fs");
-const WebSocket = require("ws")
-const config = require("./config.json");
-const revision = require('child_process').execSync('git rev-parse HEAD').toString().trim().slice(0, 6)
-const bot = new Discord.Client({
-	intents: ["GUILD_MESSAGES", "GUILD_BANS", "GUILD_MEMBERS", "GUILD_INVITES", "GUILDS"]
+
+const DBPath = path.join(__dirname, '.', 'db.json');
+
+const client = new Discord.Client({
```
Comment (ChrisChrome) [inline bot.js]: What did I say about making major changes like this
Please revert client back to bot

File: bot.js [hunk 2]
```diff
@@ -1,214 +1,202 @@
...
+	intents: ['GUILD_MESSAGES', 'GUILD_BANS', 'GUILD_MEMBERS', 'GUILD_INVITES', 'GUILDS']
 });
 
 let lastUpdate = null;
-let lastId = 0;
+let lastIdPerGuild = [];
```
Comment (ChrisChrome) [inline bot.js:39]: Okay this one was actually a good idea, no problems here lmao

**Review by ChrisChrome — CHANGES_REQUESTED** [review]
The latest version of this PR does NOT work properly, and scam URLs aren't detected

File: bot.js (hunk 1 above)
Comment (greysilly7) [reply bot.js]: A var rename isnt a major change smh. BUT THIS WHOLE PR IS A MAJOR CHANGE :)

**Pull request merged**